dsf-bpe/dsf-bpe-server/src/main/java/org/highmed/dsf/bpe/listener/CallActivityListener.java

```diff
@@ -0,0 +1,14 @@
+package org.highmed.dsf.bpe.listener;
+
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.camunda.bpm.engine.delegate.ExecutionListener;
+import org.highmed.dsf.bpe.Constants;
+
+public class CallActivityListener implements ExecutionListener
+{
+	@Override
+	public void notify(DelegateExecution execution) throws Exception
+	{
+		execution.setVariable(Constants.VARIABLE_IS_CALL_ACTIVITY, true);
```

Comment:
please add a comment, why the VARIABLE_IS_CALL_ACTIVITY variable is set to true here.